docs/editors-guide/curation-workflow-overview.md

```diff
@@ -0,0 +1,63 @@
+- **Mature terms**:  Mondo is released on a monthly basis around the first of every month. After the Mondo release, the new terms are available in the [release files](https://github.com/monarch-initiative/mondo/releases).
+- Terms will be visible in [OLS](https://www.ebi.ac.uk/ols4/ontologies/mondo) approximately a few days after the release.
+
+## Obsoleting terms in Mondo
+1. Terms in Mondo may be obsoleted for various reasons. Some examples of when to obsolete and/or merge a term are:  
+
+- **Duplicate terms** (for example MONDO:0019055 mitochondrial disease was obsoleted and replaced by MONDO:0004069 'inborn mitochondrial metabolism disorder')  
+- **Out of scope**- Some terms are excluded from Mondo based on certain criteria, which are outlined in the [exclusion reason table](editors-guide/exclusion-reasons.md). For example, terms that are not truly diseases, (ie phenotype terms, such as MONDO:0007348 Colchicine resistance). For another example, see [see #503](https://github.com/monarch-initiative/mondo/issues/503).
+- **obsoleted in source**: for example, OMIM, Orphanet or GARD may retire or obsolete a term. For example, MONDO:0015173 obsolete autoimmune enteropathy type 2
+is a phenotype and not a disease: for example, MONDO:0043606 'obsolete pathologic fracture'
+
+2. If a term is a candidate for obsoletion and/or merging, it will be reported on the [GitHub issue tracker](https://github.com/monarch-initiative/mondo/issues) and labeled 'obsolete' or 'merge'. See [obsoletion candidates here](https://github.com/monarch-initiative/mondo/issues?q=is%3Aissue+is%3Aopen+label%3Aobsolete) or [merge candidates here](https://github.com/monarch-initiative/mondo/issues?q=is%3Aissue+is%3Aopen+label%3Amerge).
+3. Issues will remain open for at least two months to allow for the community to comment and bring up any objections. All obsoletions will be done via a pull request and reviewed by Mondo developers.
+4. Terms in the ontology file will be labeled as 'obsoletion candidates' and a proposed date for obsoletion will be added to the term.
+4. With every Mondo release, we will report the terms that have been obsoleted from Mondo and the new obsoletion candidates (for example, see the [2023-02-06 release notes](https://github.com/monarch-initiative/mondo/releases/tag/v2023-02-06)).
+5. When a term is tagged for obsoletion, the community has the opportunity to provide input about the potential obsoletion of the term. 
+6. More details about the obsoletion workflow is [here](https://mondo.readthedocs.io/en/latest/editors-guide/merging-and-obsoleting/).
+
+### Request to remove a term from the obsoletion list
+
+If a Mondo terms has been tagged for obsoletion but it is in use by your project or you would like it to remain available, simply comment on the GitHub ticket or contact the Mondo Curation team via email or Slack and request that it be removed from the obsoletion candidate list. The Mondo Curation team is happy to oblige all requests.
```

Comment:
I am uncomfortable with the wording here... 
If there is a term set of obsoletion, the only reason to not obsolete it is because we are "incorrect" with our reason to obsolete it (e.g. the terms are actually not duplicate, or the term is actually a disease and not a phenotype, ...)
Using a "bad" term in a project should not be a reason to not obsolete the term in Mondo (otherwise Mondo could potentially be incorrect), but we can give people more time to review their annotations OR make a case for why we should keep it in mondo.

Reply:
good point. would you like to propos revisions to this wording?

Reply:
revisions should be in the other PR I created (ie if you merge https://github.com/monarch-initiative/mondo/pull/5975), it should update this PR with the proposed revisions.

Reply:
Ah, okay, I merged that, so we should be good now. Thank you!